bitnami/appsmith/values.yaml

```diff
@@ -0,0 +1,1247 @@
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param volumePermissions.resources.limits The resources limits for the init container
+  ## @param volumePermissions.resources.requests The requested resources for the init container
+  ##
+  resources:
+    limits: {}
+    requests: {}
+  ## Init container Container Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+  ## @param volumePermissions.containerSecurityContext.runAsUser Set init container's Security Context runAsUser
+  ## NOTE: when runAsUser is set to special value "auto", init container will try to chown the
+  ##   data folder to auto-determined user&group, using commands: `id -u`:`id -G | cut -d" " -f2`
+  ##   "auto" is especially useful for OpenShift which has scc with dynamic user ids (and 0 is not allowed)
+  ##
+  containerSecurityContext:
+    runAsUser: 0
+
+## @section Other Parameters
+##
+
+## ServiceAccount configuration
+##
+serviceAccount:
+  ## @param serviceAccount.create Specifies whether a ServiceAccount should be created
+  ##
+  create: true
+  ## @param serviceAccount.name The name of the ServiceAccount to use.
+  ## If not set and create is true, a name is generated using the common.names.fullname template
+  ##
+  name: ""
+  ## @param serviceAccount.annotations Additional Service Account annotations (evaluated as a template)
+  ##
+  annotations: {}
+  ## @param serviceAccount.automountServiceAccountToken Automount service account token for the server service account
+  ##
+  automountServiceAccountToken: true
+
+## @section External MongoDB parameters
+## All of these values are only used when mongodb.enabled is set to false
+## @param externalDatabase.hosts Database hosts
+## @param externalDatabase.port Database port number
+## @param externalDatabase.username Non-root username for Appsmith
+## @param externalDatabase.password Password for the non-root username for Appsmith
+## @param externalDatabase.database Appsmith database name
+## @param externalDatabase.existingSecret Name of an existing secret resource containing the database credentials
+## @param externalDatabase.existingSecretPasswordKey Name of an existing secret key containing the database credentials
+##
+externalDatabase:
+  hosts: []
+  port: 27017
+  username: root
+  database: appsmith
+  password: ""
+  existingSecret: ""
+  existingSecretPasswordKey: ""
+
+## @section External Redis parameters
+## All of these values are only used when redis.enabled is set to false
+## @param externalRedis.host Redis host
+## @param externalRedis.port Redis port number
+## @param externalRedis.password Password for the Redis
+## @param externalRedis.existingSecret Name of an existing secret resource containing the Redis credentials
+## @param externalRedis.existingSecretPasswordKey Name of an existing secret key containing the Redis credentials
+##
+externalRedis:
+  host: ""
```

Comment:
Why doesn't it have a default value such as **ñocalhost**? Reference: https://github.com/bitnami/charts/blob/main/bitnami/airflow/values.yaml#L1529

Reply:
I'm not sure why it has localhost as the default value (does not make much sense TBH), but I think it's important that we have a validation that forces the user to set a host. If localhost is set it may lead to a faulty deployment.